python/rapidsmpf/rapidsmpf/integrations/dask/spilling.py

```diff
@@ -130,7 +148,14 @@ def spill(self, amount: int) -> int:
             on_device = self._on_device
             if on_device is not None and self._on_host is None:
                 ret = self.approx_spillable_amount()
-                self._on_host = dask_dumps(on_device)
+                self._on_host = dask_dumps(
+                    on_device,
+                    context={
+                        "stream": stream,
+                        "device_mr": device_mr,
+                        "staging_device_buffer": staging_device_buffer,
+                    },
```

Comment:
Are all these attributes serializable?

Reply:
no, but they don't have to be. AFAICT, `context` is only passed down to the local serialization function.

Reply:
Ok, I think you're right. I wasn't familiar with the purpose of `context`, thanks for confirming.